File: TechStore_Font/src/app/api/hoadonban.ts
export interface IHoaDonBan {
    id?: number;
    userId?: number;
    hoTen?: string;
    soDienThoai?: string;
    email?: string;
    diaChi?: string;
    ghiChu?: string;
    trangThai?: string;
    tongTien?: number
    giamGia?: number
    trangThaiThanhToan?: boolean;
    chiTietHoaDonBans?: {
        sanPhamId?: number;
        soLuong?: number;
        giaBan?: number;
        thanhTien?: number;
    }[] | null;
}

File: TechStore_BackEnd_API/Models/HoaDonBan.cs
using System;
using System.Collections.Generic;

namespace TechStore.Models
{
    public partial class HoaDonBan
    {
        public HoaDonBan()
        {
            ChiTietHoaDonBans = new HashSet<ChiTietHoaDonBan>();
        }

        public int Id { get; set; }
        public string? HoTen { get; set; }
        public string? SoDienThoai { get; set; }
        public string? Email { get; set; }
        public string? DiaChi { get; set; }
        public string? GhiChu { get; set; }
        public string? TrangThai { get; set; }
        public DateTime? CreateDate { get; set; }
        public int? UserId { get; set; }
        public decimal? TongTien { get; set; }
        public decimal? GiamGia { get; set; }
        public bool? TrangThaiThanhToan { get; set; }

        public virtual KhachHang? User { get; set; }
        public virtual ICollection<ChiTietHoaDonBan> ChiTietHoaDonBans { get; set; }
    }
}
